solidity/dashboard/src/css/header.less

```diff
@@ -0,0 +1,60 @@
+.header {
+    grid-area: header;
+    width: 100%;
+    z-index: 10;
+    display: flex;
```

Comment:
Could use a comment here to indicate why we're using `display: flex`.

Reply:
actually we don't need flexbox here. I'll remove 👍